src/Installers/Windows/SharedFrameworkBundle/1033/thm.wxl

```diff
@@ -7,7 +7,7 @@
   <String Id="ConfirmCancelMessage">Are you sure you want to cancel?</String>
   <String Id="ExecuteUpgradeRelatedBundleMessage">Previous version</String>
   <String Id="HelpHeader">Setup Help</String>
-  <String Id="HelpText">/install | /repair | /uninstall | /layout [\[]directory[\]] - installs, repairs, uninstalls or
+  <String Id="HelpText"><!--_locComment_text="'directory' is to be translated in both places"-->/install | /repair | /uninstall | /layout [\[]directory[\]] - installs, repairs, uninstalls or
```

Comment:
"directory" has been messed up in multiple loc handback PRs. Loc ticket related to these problematic non-translations were closed as "Won't Fix" 😦